script/plugin_webpack_directories.rb

```diff
@@ -12,9 +11,8 @@
   specs = Bundler.load.specs
 end
 
-include Foreman::WebpackAssets
-
-config = { entries: {}, paths: [] }
+config = { entries: {}, paths: [], plugins: {} }
+plugin_name_regexp = /foreman*|katello*/
```

Comment:
This does not appear to be used anywhere

Reply:
It is within the loop below